tests/testCCRing.c

```diff
@@ -15,6 +16,14 @@ void should_fail(void) {
   CU_ASSERT_EQUAL(1, 2);
 }
 
+int test array(array1, array2, ) {
+  if =
+    return 1
+  if != 
+    return 0;
+}
+
+
```

Comment:
I don't think you meant to include  this pseudo code. Make sure to `$ git diff` every time before you `$ git add` files so you know exactly what you are changing.